Docs/MANUAL.md

```diff
@@ -0,0 +1,336 @@
+
+This is also where you export information on the seed, either the public key, the private data, or the seed itself, as described in "Using a Seed" and "Exporting a Seed".
+
+### Reading the OIB
+
+Each seed is displayed with an [Object Identity Block (OIB)](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2021-002-digest.md#object-identity-block), which can be seen on both the listing and view pages. It helps you to visually identify a seed.
+
+An OIB contains the following elements:
+
+* **Lifehash.** A [methodology](https://github.com/BlockchainCommons/LifeHash) for creating an evocative visual representation of data based on Conway's Game of Life. It makes it easy to recognize a seed at a glance.
+* **Type.** An icon representing the type of data. On the listings and main views, this is a seed icon.
+* **Name.** A human-readable name for the seed. As a default, **Seed Tool** chooses an evocative bit of nonsense derived from the content of the seed itself.
+* **Digest.** An abbreviated six-character digest of the seed.
+
+The lifehash, the type, the digest, and all but the last two words in the default name should be identical anywhere that you import your seed that uses the Blockchain Commons OIB specification. That will help you to always know that your seed was accurately transmitted, and to always make sure you're working with the right seed.
+
+OIBs are also displayed for various keys derived from your seed. They use different icons for the "type" and do not include a name, as seen in "Deriving a Key".
+
+## Using a Seed
+
+The main power of **Gordian Seed Tool** is that you can permanently store your seeds there, and instead give out keys or otherwise answer requests, as needed.
+
+### Answering Requests
+
+The Blockchain Commons [`ur:crypto-request`/`ur:crypto-response` system](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2021-001-request.md) specifies how one app can request a certain type of [UR data](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md) and another app can send that requested data. **Gordian Seed Tool** is integrated with this standard: another app can request a seed or a specific derived key, and **Gordian Seed Tool** will send it (with your approval).
+
+This is accomplished via the **Scan** (qr code) feature. Select it and import a QR code through camera, Photos, or File, or else read in a `crypto-request` through the Clipboard. You will be told what seed or key is being requested, and you can choose to approve it. If you do, you'll then be given a QR code that you can scan into the other app as the `ur:crypto-response`. 
+
+The biggest use of this function is to send a very specific derived key that the other app desires. Though you can derive a few specific keys on your own, a `ur:crypto-request` allows another app access to _any_ key derived from your seed.
```

Comment:
s/Though you can derive a few specific keys on your own,/Though Seed Tool lets you derive a few common kinds of keys from your seeds,

Reply:
changed